app/ui/src/app/connections/list/list.component.ts

```diff
@@ -59,25 +59,22 @@ export class ConnectionsListComponent implements OnInit {
     const sub = this.store.delete(connection).subscribe(
       () => {
         sub.unsubscribe();
-        setTimeout(() => {
-          this.popNotification({
-            type: NotificationType.SUCCESS,
-            header: 'Delete Successful',
-            message: 'Connection successfully deleted.',
-            showClose: true
-          });
-        }, 10);
+        this.notificationService.popNotification({
+          type: NotificationType.SUCCESS,
+          header: 'Delete Successful',
+          message: 'Connection successfully deleted.'
+        });
       },
       (err: any) => {
         sub.unsubscribe();
-        setTimeout(() => {
-          this.popNotification({
-            type: NotificationType.DANGER,
-            header: 'Delete Failed',
-            message: `Failed to delete connection: ${err}`,
-            showClose: true
-          });
-        }, 10);
+        this.notificationService.message(
+          NotificationType.DANGER,
+          'Delete Failed',
+          `Failed to delete connection: ${err}`,
+          true,
+          null,
+          []
+        );
```

Comment:
We're calling both `NotificationService.popNotification()` and `NotificationService.message()` from the same code construct in order to trigger opposite messages. Perhaps we should abstract everything from `NotificationService.popNotification()` by extending its method payload.